packages/charts/src/Charts/SeverityLine/severity-line.scss

```diff
@@ -0,0 +1,61 @@
+.ins-c-severity-line {
+    --inc-c-severity-line-base:         var(--pf-global--palette--black-400);
+    --ins-c-severity-line-low:          var(--pf-global--palette--blue-200);
+    --ins-c-severity-line-moderate:     var(--pf-global--palette--gold-200);
+    --ins-c-severity-line-high:         var(--pf-global--palette--orange-200);
+    --ins-c-severity-line-critical:     var(--pf-global--palette--red-100);
+
+    display: inline-flex;
+}
+
+.ins-c-severity-line {
+    .ins-l-title {
+        width: 115px;
+        span {
+            color: var(--inc-c-severity-line-base);
```

Comment:
Looking at the marvel mockups, the title here seems a little too light. I'd just double check this passes with UX.

Reply:
I update this to the value they actually specified, so should be good now 👍